doc/configuration.md

```diff
@@ -96,6 +97,16 @@ itself. It defaults to `false`.
 verbose_trace: true
 ```
 
+### `chain_stack_traces`
+
+This boolean field controls whether or not stack traces are chained. 
+Disabling stack trace chaining will improve performance at the cost of
+debuggability. 
```

Comment:
Consider linking [the stack chain docs](https://github.com/dart-lang/stack_trace/blob/master/README.md#stack-chains) here so people have a better idea of what debuggability they get.